Source/Embeddings.Processing/FailedDeleteEmbedding.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.Runtime.Embeddings.Store;
+
+namespace Dolittle.Runtime.Embeddings.Processing
+{
+    /// <summary>
+    /// Exception that gets thrown when a failure occurs while deleting an embedding.
+    /// </summary>
+    public class FailedDeleteEmbedding : Exception
```

Comment:
Same thing, somewhat strange name...

Reply:
Same as above. Do you have any suggestions?